source/configure-components-with-javascript/index.html.md.erb

```diff
@@ -83,7 +83,7 @@ Read the [JavaScript API Reference](/javascript-api-reference/) to see what conf
 You can pass configuration for components when initialising GOV.UK Frontend using the `initAll` function. You can do this by including key-value pairs of camel-cased component names and configuration objects. This is the same method you would use to pass them when creating an instance of the component. For example:
 
 ```javascript
-window.GOVUKFrontend.initAll({
+initAll({
```

Comment:
What contexts will `initAll` be available in? Does this need an import statement to make it clear?

Reply:
Thanks for spotting this @domoscargin, makes me feel better

I had the same worries but noticed we (mostly) don't add imports in other examples

But otherwise, we'd have to weigh up three approaches now:

1. UMD via `const { initAll } = require('govuk-frontend')`
1. ES modules (bundler) via `import { initAll } from 'govuk-frontend'`
1. ES modules (browser) via `import { initAll } from '/javascripts/govuk-frontend-5.0.0.min.js'`

One to improve in future?

Reply:
Yeah, that sounds sensible!